lib/stat_tracker.rb

```diff
@@ -257,4 +257,21 @@ def most_tackles(season)
 
     return team_with_most_tackles.team_name
   end
+
+  def fewest_tackles(season)
+    season_data = find_season(season)
+    team_tackles = Hash.new(0)
+  
+    season_data.game_teams.each do |game_team|
+      team_id = game_team.team_id
+      tackles = game_team.tackles
+  
+      team_tackles[team_id] += tackles
+    end
```

Comment:
245-253 and 262-270 could be DRY'd up into a helper method.
The resulting helper method could even be put into the Season class-- I can add something in Season that organizes team tackle data for the season, so that all we need to do from StatTracker is find that season, then call on that season's tackle stats. Thoughts?

Reply:
I'm all for that implementation, does that mean I would have to wait on you to finish the season class for that helper method?

Reply:
I've put in a PR with the helper method ready to go! I will continue to spend time on further refactoring as well